django_mongodb/functions.py

```diff
@@ -99,7 +160,9 @@ def extract(self, compiler, connection):
 
 def func(self, compiler, connection):
     lhs_mql = process_lhs(self, compiler, connection)
-    operator = MONGO_OPERATORS.get(self.__class__, self.function.lower())
+    operator = MONGO_OPERATORS.get(
+        self.__class__, (self.extra["function"] if self.function is None else self.function).lower()
```

Comment:
`self.extra["function"]` may not be a bad idea to support, but I wonder if it works for any of the existing tests which often use SQL functions rather than MongoDB aggregation names. I see it's used here:
 `aggregation/tests.py:        raw_books_count = Func("book", function="COUNT")`
but this test is failing for another reason, I believe.

If there aren't any test failures without it, I'd remove this change from this PR.